qingstor/qsctl/commands/presign.py

```diff
@@ -56,6 +56,15 @@ def generate_presign_url(cls):
             sys.exit(-1)
         cls.validate_bucket(bucket)
         current_bucket = cls.client.Bucket(bucket, cls.bucket_map[bucket])
+
+        current_acl = current_bucket.get_acl()
+        for v in current_acl["acl"]:
+            # go through the whole list
+            if "QS_ALL_USERS" == v["grantee"]["name"] and "FULL_CONTROL" == v["permission"]:
+                obj_url = "https://" + cls.bucket_map[bucket] + ".qingstor.com/" + bucket + "/" + prefix
```

Comment:
1. Do not hard code `https` and `qingstor.com` here.

1. Use the following code will be better for understanding:

    ```python
    obj_url = "{protocol}://{bucket_name}.{zone}.{host}/{object_key}".format(
        protocol="https",
        bucket_name="mybucket",
        zone="pek3a",
        host="qingstor.com",
        object_key="test_object"
    )
    ```
1. `obj_url` is not a good name, wo should find a better one